src/gt4py/backend/gt_backends.py

```diff
@@ -368,9 +368,7 @@ def visit_AxisBound(self, node: gt_ir.AxisBound) -> Tuple[int, int]:
         elif node.level == gt_ir.LevelMarker.END:
             level = len(self.k_splitters) + 1
         else:
-            assert isinstance(node.level, gt_ir.VarRef)
-            assert len(node.level.index) == 1
-            level = self.k_splitters.index((node.name, node.index[0]))
```

Comment:
This is what had to get removed to satisfy mypy.

Reply:
:+1: